webapp/app/partials/perf/comparectrl.html

```diff
@@ -19,10 +19,12 @@
       <h4>Compare all data for revision {{newProject}}: {{newRevision}} to {{originalProject}}: {{originalRevision}}</h4>
       <p class="help-block">Hover over each entry for more options.</p>
       <table class="table">
-        <tbody>
-          <tr ng-class="{'subtest-header': compareResult.headerColumns==2, 'subtest-empty': compareResult.isEmpty && !compareResult.headerColumns==2, 'subtest-empty': (hideMinorChanges && compareResult.isMinor)}" ng-repeat="compareResult in compareResults">
-            <td colspan="{{compareResult.headerColumns}}">{{compareResult.name}}</td>
-            <td ng-class="{'subtest-empty': compareResult.headerColumns==2}"><a ng-href="{{compareResult.detailsLink}}">Details</a></td>
+        <tbody ng-repeat="testName in testList">
+          <tr class="subtest-header">
+            <td>{{testName.replace('summary ', '')}}</td><td>Old Geomean</td><td>Old StdDev</td><td>New Geomean</td><td>New StdDev</td><td>Delta</td><td>Delta %</td>
```

Comment:
Putting a .replace() inside the template is pretty gross. Is there no way we could generate the required string in the code somehow? I guess the best thing would be to store the name stripped of keywords like "summary" and just use that here,